cram/cram_io.c

```diff
@@ -2080,7 +2080,7 @@ int cram_compress_block2(cram_fd *fd, cram_slice *s,
                     1.04, // 6  arithpr (O0)
                     1.05, // 7  fqz
                     1.05, // 8  tok3 (rans)
-                    9, 9, // 9,10 reserved
+                    1.00, 1.00, // 9,10 reserved
```

Comment:
If we're redefining them elsewhere anyway, then probably something like 0 would make more sense as an explicit nul value instead of 1.00.

Reply:
I didn't use 0 because there's some [scaling code](https://github.com/samtools/htslib/blob/2d1e9549/cram/cram_io.c#L2115) that subtracts one from the values here before multiplying them, and I didn't want to end up with a negative value.